ui/src/main/webapp/src/app/shared/table/data-table.component.ts

```diff
@@ -0,0 +1,47 @@
+import {OrderDirection, SortConfiguration} from "../sort/sorting.service";
+import {
+    ChangeDetectionStrategy, Component, ContentChild, EventEmitter, Input, Output,
+    TemplateRef
+} from "@angular/core";
+
+@Component({
+    selector: 'wu-data-table',
```

Comment:
Before we merge this - could we please rename it to `wu-filtered-table`? I think that matches what it is. All tables contain data :)

Reply:
+1

Reply:
Hmm but that would imply it is always filtered. But what if filter is optional? `wu-filtered-table` without using filter would look weird.

Reply:
Oh, that is a good point. I think that you are right then, wu-data-table is better.